sycl/doc/Array_Kernel_Parameters.md

```diff
@@ -0,0 +1,359 @@
+<h2>Array Parameters of SYCL Kernels</h2>
+
+<h3>Introduction</h3>
+
+This document describes the changes to support passing arrays to SYCL kernels and special treatment of Accessor arrays. The following cases are handled:

```

Comment:
Shorter lines are easier to review, specially with my punch-card reader... :-)

Reply:
Done.